.github/ISSUE_TEMPLATE/lsp_bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: Language server client bug report
 about: Report a built-in lsp problem in Nvim
 title: ''
-labels: bug, lsp
+labels: '0.kind: bug', topic:lsp
```

Comment:
If we use prefixes already, shouldn't it be `'topic: lsp'` as well?

Reply:
actually I ponder if we should prefix those with `topic:` ? I mostly wanted bugs to appear first which is accomplished with the leading `0.`. As for the missing space I agree.